pom.xml

```diff
@@ -88,34 +91,99 @@
             <groupId>com.github.onsdigital</groupId>
             <artifactId>dp-logging</artifactId>
             <version>${dp.logging.version}</version>
         </dependency>
-        <dependency>
-            <groupId>com.github.onsdigital</groupId>
-            <artifactId>dp-jwt-verifier-java</artifactId>
-            <version>0.5.0</version>
-        </dependency>
+
     </dependencies>
 
     <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <version>${maven.surefire.plugin.version}</version>
+                </plugin>
+                <plugin>
+                    <groupId>org.sonatype.ossindex.maven</groupId>
+                    <artifactId>ossindex-maven-plugin</artifactId>
+                    <version>${maven.sonatype.ossindex.plugin.version}</version>
+                    <executions>
+                        <execution>
+                            <id>audit-dependencies-critical</id>
+                            <phase>validate</phase>
+                            <goals>
+                                <goal>audit</goal>
+                            </goals>
+                            <!-- configuration for mvn validate -->
+                            <configuration>
+
+                                <!-- if CVSS >= 9.0 (critical) then ERROR else WARN -->
+                                <fail>true</fail>
+                                <cvssScoreThreshold>9.0</cvssScoreThreshold>
+                            </configuration>
+                        </execution>
+                    </executions>
+                    <!-- configuration for mvn ossindex:audit -->
+                    <configuration>
+                        <!-- if CVSS >= 7.0 (high or critical) then ERROR else WARN -->
+                        <fail>true</fail>
+                        <cvssScoreThreshold>7.0</cvssScoreThreshold>
+                        <excludeCoordinates>
+                            <exlude>
+                                <groupId>com.fasterxml.jackson.core</groupId>
+                                <artifactId>jackson-core</artifactId>
+                                <version>2.14.1</version>
+                            </exlude>
+                        </excludeCoordinates>
+                    </configuration>
+                </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-checkstyle-plugin</artifactId>
+                    <version>3.1.2</version>
+                    <dependencies>
+                        <dependency>
+                            <groupId>com.puppycrawl.tools</groupId>
+                            <artifactId>checkstyle</artifactId>
+                            <version>9.3</version>
+                        </dependency>
```

Comment:
Would you be able to explain the reason this is required? It seems to run fine without it

Reply:
https://github.com/search?q=org%3AONSdigital+com.puppycrawl.tools&type=code